### DISCUSSION TITLE
Being able to select all text in a textbox using Ctrl+A

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#30808] [Ideas] [answered]
Just a small QoL thing, would be nice to be able to select everything in a text field with Ctrl+A, in order to write over the previous text 👍 
Thanks!

**Comment by Joehuu** [comment]
Weird, this is already a thing. Maybe something on your system is absorbing that input.

**Reply by vatei** [reply]
... Lmao I know
I'm on AZERTY that's why it's not working and I didn't notice this was a thing
Ctrl+q works

**Reply by Joehuu** [reply, marked as answer]
Will be fixed in https://github.com/ppy/osu-framework/pull/5790.
